tools/sources-R5.0.lst

```diff
@@ -7,3 +7,7 @@ vrouter Juniper contrail-vrouter R5.0
 common Juniper contrail-common R5.0
 analytics Juniper contrail-analytics R5.0
 apiClient Juniper contrail-api-client R5.0
+webuiThirdParty Juniper contrail-webui-third-party R5.0
+webCore Juniper contrail-web-core R5.0
+generateds Juniper contrail-generateds master
```

Comment:
Is this repo really necessary ?

Because generateds code has been moved in contrail-api-client repo, and this is probably why there is no R5.0 branch in this repo

Reply:
Well, the build scripts i was looking at were using it - i can have a look and see if I can get rid of it and use `contrail-api-client` instead

Reply:
Wasn't necessary! I removed it.